ios/Classes/AblyFlutterStreamHandler.m

```diff
@@ -32,39 +32,39 @@ - (void) startListening:(AblyFlutterMessage *const)message emitter:(FlutterEvent
     NSString *const eventName = eventMessage.eventName;
     @try {
         if ([AblyPlatformMethod_onRealtimeConnectionStateChanged isEqual: eventName]) {
-            ARTRealtime* realtime = [_ably realtimeFrom:handle];
-            listener = [realtime.connection on: ^(ARTConnectionStateChange * const stateChange) {
+            ARTRealtime *const realtime = [_instanceStore realtimeFrom:handle];
+            listener = [realtime.connection  on: ^(ARTConnectionStateChange * const stateChange) {
                 emitter(stateChange);
             }];
         } else if ([AblyPlatformMethod_onRealtimeChannelStateChanged isEqual: eventName]) {
             NSAssert(eventMessage.message, @"event message is missing");
             NSMutableDictionary<NSString *, NSObject *>* eventPayload = eventMessage.message;
-            ARTRealtime* realtime = [_ably realtimeFrom:handle];
+            ARTRealtime *const realtime = [_instanceStore realtimeFrom:handle];
             
-            NSString *channelName = (NSString*) eventPayload[TxTransportKeys_channelName];
-            ARTRealtimeChannel *channel = [realtime.channels get:channelName];
+            NSString *const channelName = (NSString*) eventPayload[TxTransportKeys_channelName];
```

Comment:
casting to `NSString` on the right side is redundant if declared expected type on left side
```suggestion
            NSString *const channelName = eventPayload[TxTransportKeys_channelName];
```